src/test/java/com/ably/kafka/connect/DefaultAblyBatchClientTest.java

```diff
@@ -65,4 +77,95 @@ public void testGroupMessagesByChannel() throws AblyException {
         assertTrue(topic2Messages.get(1).data.equals("test4"));
 
     }
+
+    @Test
+    public void testParseAblyBatchAPIResponse() throws AblyException {
+        final String STATIC_CHANNEL_NAME = "channel_#{topic}";
+        final ChannelSinkConnectorConfig connectorConfig = new ChannelSinkConnectorConfig(Map.of("channel",
+                STATIC_CHANNEL_NAME, "client.key", "test-key", "client.id", "test-id"));
+        final ConfigValueEvaluator configValueEvaluator = new ConfigValueEvaluator();
+        final ChannelConfig channelConfig = new DefaultChannelConfig(connectorConfig);
+        final ChannelSinkMapping channelSinkMapping = new DefaultChannelSinkMapping(configValueEvaluator, channelConfig);
+        final MessageSinkMapping messageSinkMapping = new DefaultMessageSinkMapping(connectorConfig, configValueEvaluator);
+        DefaultAblyBatchClient client = new DefaultAblyBatchClient(connectorConfig, channelSinkMapping,
+                messageSinkMapping, configValueEvaluator);
+
+
+        HttpPaginatedResponse response = new HttpPaginatedResponse() {
+            @Override
+            public JsonElement[] items() {
+                return new JsonElement[0];
+            }
+
+            @Override
+            public HttpPaginatedResponse first() throws AblyException {
+                return null;
+            }
+
+            @Override
+            public HttpPaginatedResponse current() throws AblyException {
+                return null;
+            }
+
+            @Override
+            public HttpPaginatedResponse next() throws AblyException {
+                return null;
+            }
+
+            @Override
+            public boolean hasFirst() {
+                return false;
+            }
+
+            @Override
+            public boolean hasCurrent() {
+                return false;
+            }
+
+            @Override
+            public boolean hasNext() {
+                return false;
+            }
+
+            @Override
+            public boolean isLast() {
+                return false;
+            }
+        };
+        response.statusCode = 5000;
+
+        assertEquals(DefaultAblyBatchClient.ABLY_BATCH_RESPONSE.FAILURE,
+                client.parseAblyBatchAPIResponse(response));
+
+        response.statusCode = 40000;
+        assertEquals(DefaultAblyBatchClient.ABLY_BATCH_RESPONSE.FAILURE,
+                client.parseAblyBatchAPIResponse(response));
```

Comment:
The response `statusCode` will only ever be a real HTTP status code, it's `errorCode` that will typically be a 5 digit number. Can we change these to be realistic values to avoid confusion, such as `500` and `400`?

Reply:
When I was testing the status Code was 0